lib/rubotnik/helpers.rb

```diff
@@ -38,7 +41,7 @@ def text_message?
     end
 
     def message_contains_location?
-      @message.attachments && @message.attachments.first['type'] == 'location'
+      !@message.attachments.nil? && @message.attachments.first['type'] == 'location'
```

Comment:
This method returned `nil` instead of `false` if `message.attachments` was `nil` - changed that to return true/false